src/backend/dwarf.c

```diff
@@ -78,6 +80,23 @@ static Outbuffer  *reset_symbuf;        // Keep pointers to reset symbols
 static char __file__[] = __FILE__;      // for tassert.h
 #include        "tassert.h"
 
+/***********************************
+ * Determine if generating a eh_frame with full
+ * unwinding information.
+ * This decision is done on a per-function basis.
+ * Returns:
+ *      true if unwinding needs to be done
+ */
+bool doUnwindEhFrame()
+{
+    /* FreeBSD fails when having some frames as having unwinding info and some not.
+     * (It hangs in unittests for std.datetime.)
+     * g++ on FreeBSD does not generate mixed frames, while g++ on OSX and Linux does.
+     */
+    assert(!(usednteh & ~(EHtry | EHcleanup)));
+    return (usednteh & (EHtry | EHcleanup)) || (config.exe & (EX_FREEBSD | EX_FREEBSD64));
```

Comment:
Yes, but I still don't understand why it's green in master https://auto-tester.puremagic.com/platform-history.ghtml?projectid=1&os=FreeBSD_64_64